curriculum/challenges/english/24-b1-english-for-developers/learn-common-phrasal-verbs-and-idioms/68487cc8ff08cd68db3c6d27.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 68487cc8ff08cd68db3c6d27
+title: Task 8
+challengeType: 19
+dashedName: task-8
+---
+
+<!-- (Audio) Brian: Let's start by phasing out the old clause about mandatory in-office days. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What does Brian believe they should do first?
+
+## --answers--
+
+Add a new clause about flexible hours.
+
+### --feedback--
+
+Brian doesn't talk about adding new items to the policy.
+
+---
+
+Decide who will approve the updated policy.
+
+### --feedback--
+
+Brian does not mention an approval.
+
+---
+
+Remove the rule that requires people to work from the office.
+
+---
+
+Ask the team how many days they want to work in the office.
+
+### --feedback--
+
+Brian does not mention asking for input.
+
+## --video-solution--
+
+3
+
+# --explanation--
+
+`To phase out` means to slowly remove or stop using something over time. For example:
```

Comment:
If `phase out` is a new phrase, I suggest using a type 22 task to teach it first.

Reply:
Let us test how this goes with a type-19. I'll make the change in an upcoming PR.